### ISSUE TITLE
[Table] text-overflowed cells should have a tooltip to show the full text

### ISSUE THREAD

**Issue by Chris-Xie** [Tradeshift/tradeshift-ui#543]
Choose the section that applies to you and use the template to help us figure out how we can help you.

----------------------------
#### Tradeshift UI version affected
v9.2.0

#### Expected Behavior
It would be nice to have a tooltip (can be easily implemented with a `title` attribute) showing the full text when you mouse over that cell that have text clipped.
#### Screenshots (optional)
<img width="1340" alt="screen shot 2018-02-23 at 10 09 51" src="https://user-images.githubusercontent.com/5295157/36574657-b75effb6-1881-11e8-8683-aa3db8914195.png">

---------------------------

**Comment by wiredearp** [comment]
Good idea - even if it is unfortunately very expensive to detect, whether or not the cell actually has overflow (ellipsis is present). With some luck, we already read the involved properties `offsetWidth` and `scrollWidth` somewhere in the code and then this calculation can be made "for free". Thanks for the request!
